okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/OkHttpClients.java

```diff
@@ -54,6 +54,16 @@
      */
     private static final ExecutorService executionExecutor = Tracers.wrap(new Dispatcher().executorService());
 
+    /**
+     * Shared dispatcher with static executor service.
```

Comment:
super nit: can use `/** One-line javadoc. */`

Reply:
sure. I just followed the style of the above field's javadoc, which admittedly just barely doesn't fit on 1 line.